dockers/base-cuda/Dockerfile

```diff
@@ -114,6 +114,10 @@ RUN \
    cat ./requirements/horovod.txt && \
    cmake --version && \
    pip install --no-cache-dir -r ./requirements/horovod.txt && \
+   horovodrun --check-build && \
+   pip uninstall -y horovod && \
+   pip install --no-cache-dir -r ./requirements/horovod.txt && \
+   horovodrun --check-build && \
```

Comment:
Reinstalling it somehow fixes the error...🤔

<details><summary>result here</summary>

```console
root@b43ef4616cac:/# horovodrun --check-build
Horovod v0.24.2:

Available Frameworks:
    [ ] TensorFlow
    [X] PyTorch
    [ ] MXNet

Available Controllers:
    [ ] MPI
    [ ] Gloo

Available Tensor Operations:
    [ ] NCCL
    [ ] DDL
    [ ] CCL
    [ ] MPI
    [ ] Gloo
root@b43ef4616cac:/# python -c "import horovod.torch"
Extension horovod.torch has not been built: /usr/local/lib/python3.7/dist-packages/horovod/torch/mpi_lib/_mpi_lib.cpython-37m-x86_64-linux-gnu.so not found
If this is not expected, reinstall Horovod with HOROVOD_WITH_PYTORCH=1 to debug the build error.
Warning! MPI libs are missing, but python applications are still available.
root@b43ef4616cac:/# pip uninstall -y horovod
...
root@b43ef4616cac:/# pip install --no-cache-dir horovod
...
root@b43ef4616cac:/# horovodrun --check-build
Horovod v0.24.2:

Available Frameworks:
    [ ] TensorFlow
    [X] PyTorch
    [ ] MXNet

Available Controllers:
    [ ] MPI
    [X] Gloo

Available Tensor Operations:
    [X] NCCL
    [ ] DDL
    [ ] CCL
    [ ] MPI
    [X] Gloo
root@b43ef4616cac:/# python -c "import horovod.torch"
root@b43ef4616cac:/#
```

</details>

Reply:
@akihironitta at this point, anything that makes it work is good enough :)) 
Did you push the image? You may need to retrigger the GPU CI job to pick up the changes

Reply:
Working on it!